Mathlib/SetTheory/Game/Ordinal.lean

```diff
@@ -150,8 +157,41 @@ noncomputable def toPGameEmbedding : Ordinal.{u} ↪o PGame.{u} where
 /-- Converts an ordinal into the corresponding game. -/
 noncomputable abbrev toGame (o : Ordinal) : Game := ⟦o.toPGame⟧
 
+/-- The order embedding version of `toGame`. -/
+@[simps]
+noncomputable def toGameEmbedding : Ordinal.{u} ↪o Game.{u} where
+  toFun := Ordinal.toGame
+  inj' a b := by simp
+  map_rel_iff' := @toPGame_le_iff
+
+@[simp]
+theorem toGame_zero : toGame 0 = 0 :=
+  game_eq toPGame_zero
+
+@[simp]
+theorem toGame_one : toGame 1 = 1 :=
+  game_eq toPGame_one
+
+theorem toGame_injective : Function.Injective toGame :=
+  toGameEmbedding.injective
+
+@[simp]
+theorem toGame_lf_iff {a b : Ordinal} : Game.LF (toGame a) (toGame b) ↔ a < b :=
```

Comment:
Not being able to use `⧏` is really annoying... This was probably discussed before, but is the convention to avoid `open`ing both `Game` and `PGame` at the same time? I guess in this case adding `PGame.` in a bunch of places isn't worth one use of `⧏`, but I'm a bit uncomfortable with not always using notation.

Reply:
In truth I'm not actually sure we should have `LF`. Originally, this predicate had to be defined alongside the usual `≤` relation through mutual induction. But this is no longer needed thanks to Junyan Xu's game induction relation. So really `LF x y` is the same as `not y ≤ x`, which has no sort of dedicated notation on any other type.

Reply:
I assume this is a discussion for future developments, and shouldn't be seen as an obstacle to merging this PR?

Reply:
Yes this is just the first time I'm seeing this part of the library and was wondering about it. Definitely won't be changed right now.